### DISCUSSION TITLE
[Poll] How are you currently using Buefy today?

### DISCUSSION THREAD

**Discussion by wesdevpro** [buefy/buefy#3990] [Polls]
[Poll here](https://docs.google.com/forms/d/e/1FAIpQLSc16dRzemCEYQIm_y-OvubsKWIZbvkcFK82xCKVqUceP4DCaw/viewform?usp=sf_link)

**Comment by rcoundon** [comment]
We could do with the ability to select multiple options.

**Reply by wesdevpro** [reply]
For some reason, there is no option for GitHub polls to allow multiple options selection. https://github.com/orgs/community/discussions/52039

**Reply by wesdevpro** [reply]
@rcoundon I have moved the poll to [Google Forms](https://forms.gle/PB57CBKQic1Axnzd9)
